docs/language/ql-training-rst/index.rst

```diff
@@ -10,17 +10,5 @@ QL training and variant analysis examples
    :maxdepth: 1
    :hidden:
 
-   ./cpp/intro-ql-cpp
-   ./cpp/bad-overflow-guard
-   ./cpp/program-representation-cpp
-   ./cpp/data-flow-cpp
-   ./cpp/snprintf
-   ./cpp/global-data-flow-cpp
-   ./cpp/control-flow-cpp
-   
-.. toctree::
-   :glob:
-   :maxdepth: 1
-   :hidden:
-
-   ./java/intro-ql-java
\ No newline at end of file
+   ./**/*
+   template
```

Comment:
I've changed this slightly so that all .rst files in ql-training-rst and its subfolders are automatically included in the toctree (unless their pattern is excluded in conf.py)